contrib/debian/examples/bitcoin.conf

```diff
@@ -94,6 +104,14 @@
 #rpcsslcertificatechainfile=server.cert
 #rpcsslprivatekeyfile=server.pem
 
+# Transaction Fee Changes in 0.10.0
```

Comment:
Rather than group by version it was added in, it makes sense to group by topic IMO.